vsintegration/src/FSharp.Editor/LanguageService/WorkspaceExtensions.fs

```diff
@@ -226,6 +227,7 @@ type Project with
         let documents = this.Documents |> Seq.filter (fun document -> not (canSkipDocuments.Contains document.FilePath))
 
         if this.IsFastFindReferencesEnabled then
+            TelemetryReporter.reportEvent "fastfindreferences" []
```

Comment:
This will report an event for each project in a solution every time someone does find references or rename. 

I would probably do just one event for solution. And maybe even separate for finding and renaming. I can make the changes and push them here. Or do a separate PR. 

I also thought about reporting whenever someone does it with the feature turned off, so we see the ratio of how much it's used compared to total. But that might just be a waste of space, not sure how much we're allowed to use.

Reply:
Discussed offline, we'll be done separately, for now added a guaranteed event "languageservicestarted".